src/monthly-giving-circle.ts

```diff
@@ -1,22 +1,104 @@
 /* eslint-disable no-debugger */
 
-import { LitElement, html } from 'lit';
+import { LitElement, html, TemplateResult, nothing } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 
 import './welcome-message';
 import './presentational/mgc-title';
+import './receipts';
+import type { IauxMgcReceipts } from './receipts';
+import './presentational/iaux-button';
+
+export type anUpdate = {
+  message: string;
+  status: 'success' | 'fail';
+  donationId: string;
+};
 
 @customElement('iaux-monthly-giving-circle')
 export class MonthlyGivingCircle extends LitElement {
   @property({ type: String }) patronName: string = '';
 
+  @property({ type: Array }) receipts = [];
+
+  @property({ type: Array }) updates: anUpdate[] = [];
+
+  @property({ type: String, reflect: true }) viewToDisplay:
+    | 'welcome'
+    | 'receipts' = 'welcome';
+
   protected createRenderRoot() {
     return this;
   }
 
+  get receiptListElement(): IauxMgcReceipts {
+    return this.querySelector('iaux-mgc-receipts') as IauxMgcReceipts;
+  }
+
+  updateReceived(update: anUpdate) {
+    this.receiptListElement.emailSent({
+      id: update.donationId,
+      emailStatus: update.status,
+    });
+    this.updates.unshift(update);
+  }
+
+  get showReceiptsCTA(): TemplateResult {
+    return html`
+      <iaux-button
+        class="link"
+        style="--button-padding: 0;"
+        .clickHandler=${() => {
+          this.viewToDisplay = 'receipts';
+          this.dispatchEvent(new CustomEvent('ShowReceipts'));
+        }}
+      >
+        View recent donation history
+      </iaux-button>
+    `;
+  }
+
   protected render() {
+    if (this.viewToDisplay === 'receipts') {
```

Comment:
Similar suggestion about breaking up this render method to make it easier to read. I didn't notice there are two different versions here that you can display. ie:
```
if (this.viewToDisplay === 'receipts') {
  ${this.receiptsView}
} else {
  ${this.welcomeView}
}
```